generator/template/process.go

```diff
@@ -169,9 +172,40 @@ func processTableSQLBuilder(fileTypes, dirPath string,
 
 		err = utils.SaveGoFile(tableSQLBuilderPath, tableSQLBuilder.FileName, text)
 		throw.OnError(err)
+
+		generatedBuilders = append(generatedBuilders, tableSQLBuilder)
+	}
+
+	if len(generatedBuilders) > 0 {
+		generateSetSchema(dirPath, fileTypes, schemaMetaData, generatedBuilders)
 	}
 }
 
+func generateSetSchema(dirPath, fileTypes string, schemaMetadata metadata.Schema, builders []TableSQLBuilder) {
+
+	basePath := path.Join(dirPath, builders[0].Path)
+	err := utils.EnsureDirPath(basePath)
+	throw.OnError(err)
```

Comment:
Caller function already insured directory exist.

Reply:
Resolved in 04c1a51